spark-cassandra-connector/src/test/scala/com/datastax/driver/core/RowMock.scala

```diff
@@ -0,0 +1,114 @@
+package com.datastax.driver.core
```

Comment:
I don't think it would be much simpler. I think it would be opposite because I would need to mock `getBytesUnsafe` and `isNull` in every case. However, I added inheritance from `AbstractGettableData` so that there are much less methods.